webapp/src/cardFilter.ts

```diff
@@ -50,19 +50,19 @@ class CardFilter {
             if (filter.values.length < 1) {
                 break
             }		// No values = ignore clause (always met)
-            return (filter.values.find((cValue) => cValue === value) !== undefined)
+            return (filter.values.find((cValue) => (Array.isArray(value) ? value.includes(cValue) : cValue === value)) !== undefined)
         }
         case 'notIncludes': {
             if (filter.values.length < 1) {
                 break
             }		// No values = ignore clause (always met)
-            return (filter.values.find((cValue) => cValue === value) === undefined)
+            return (filter.values.find((cValue) => (Array.isArray(value) ? value.includes(cValue) : cValue === value)) === undefined)
         }
         case 'isEmpty': {
-            return !value
+            return value.length <= 0
```

Comment:
What would happen with a checkbox, for example?

Reply:
value here according to typescript can only be a `string | string[]`.  Checkbox option does not show up. It's impossible to filter by checkbox.

![image](https://user-images.githubusercontent.com/17804942/121903246-1a49d700-ccf6-11eb-9f73-b01681c90253.png)

You can see prior to my changes, only select options showed up in that dropdown and I've added multiselect.

Reply:
:+1: